### DISCUSSION TITLE
Double check: Export functions in v3.0

### DISCUSSION THREAD

**Discussion by jchen1992** [esuxiao/mmeta#14] [Q&A] [answered]
Just check with you. In new version 3.0, the new export functions are:
SingleTable.create,    SingleTable.modelFit,
SingleTable.summary,  SingleTable.plot,
MultipeTables.create,   MultipeTables.modelFit,
MultipeTables.summary,  MultipeTables.plot

Are there any further functions should be exported?

**Comment by esuxiao** [comment, marked as answer]
No further functions need to exported.
